src/app/pages/signing/image-modal/image-modal.component.ts

```diff
@@ -30,21 +27,26 @@ export class ImageModalComponent {
     this.imageModal.hide();
   }
 
-  fileNames(fileInput: any) {
-    this.selectedImagesNames = [];
-    this.selectedImages = [];
+  onSelectFiles(e) {
+    const files = e.target.files;
```

Comment:
Can you check to see if this triggers a warning/error in code inspection? I can't check rn

Reply:
Just inspected `image-modal.component.ts`. No suspicious code found.